vm-setup/roles/v1aX_integration_test/tasks/generate_templates.yml

```diff
@@ -32,8 +32,10 @@
       - workers
 
   - name: Generate templates
+    vars:
+      clusterctl_var: "{{ 'generate' if CAPI_VERSION == 'v1beta1' else 'config' }}"
```

Comment:
Will this work for CAPI v1alpha3? Because I had the PR to change this, and generate is not yet there in v1alpha3 it seems https://github.com/metal3-io/metal3-dev-env/pull/847

Reply:
It worked because CAPI v1a4 is `test-v1a4-integration`and it passed, can you check attached screenshot above? Ah, I did not know that you have 847

Reply:
It is normal because you replaced it completely and that is a problem for CAPI v1a3 compared here it is made conditional based on the CAPI API version, but not sure how it passed v1b1 integration test for you(under the hood it is v1a5 still currently)

Reply:
What is not clear to me is, it didn't pass for me in v1a3 of CAPI, because clusterctl version didn't have generate command at that time, which is expected, but somehow it passed here, what surprises me

Reply:
sorry I misread the conditional. This looks good